msprime/trees.py

```diff
@@ -637,7 +663,7 @@ def get_sample_size(self):
     def draw(
             self, path=None, width=200, height=200, show_times=False,
             show_mutation_labels=False, show_internal_node_labels=True,
-            show_leaf_node_labels=True):
+            show_leaf_node_labels=True, node_colours=[]):
```

Comment:
``branch_colours`` is a better name here, as ``node_colours`` might be used to colour the nodes themselves at some point. Also a default value of None is safer here as it allows more freedom to expand the API with mutually exclusive options at some later point.